pkgs/development/python-modules/numpy/2.nix

```diff
@@ -59,15 +59,15 @@ let
 in
 buildPythonPackage rec {
   pname = "numpy";
-  version = "2.3.2";
+  version = "2.3.3";
   pyproject = true;
 
   disabled = pythonOlder "3.11";
 
   src = fetchPypi {
```

Comment:
Out of curiosity, what's the reason we don't use fetchFromGitHub?

Reply:
No reason that I'm aware of.